docs/CHANGELOG.asciidoc

```diff
@@ -56,6 +57,264 @@ Review important information about the {kib} 8.x releases.
+* Adds the KQL bar to embeddables ({kibana-pull}171016[#171016]).
+* Enables the average mobile app launch time panel ({kibana-pull}170773[#170773]).
+* Enables the mobile most launches panel ({kibana-pull}168925[#168925]).
+* Improves the Differential TopN functions grid view ({kibana-pull}170008[#170008]).
+Cases::
+* Users can copy to the clipboard the hashes of files uploaded to cases ({kibana-pull}172450[#172450]).
+* Allow users to configure which columns are displayed in the cases list including custom fields ({kibana-pull}170950[#170950]).
+* Adds a new sub-feature privilege to control user access to the cases settings ({kibana-pull}170635[#170635]).
+Dashboard::
+* Adds Links to the Visualization library ({kibana-pull}170810[#170810]).
+Discover::
+* Adds a field tokens column in the grid header ({kibana-pull}167179[#167179]).
+* Enables the addition of columns from the document viewer ({kibana-pull}171083[#171083]).
+* Adds field search via wildcards in the document viewer ({kibana-pull}168616[#168616]).
+* Improves search for field names by handling spaces like wildcards ({kibana-pull}168381[#168381]).
+* Updates mapping conflict popover with types list ({kibana-pull}169855[#169855]).
+* On search source error, show 'view details' action that opens request in inspector ({kibana-pull}170790[#170790]).
+* Adds an Unsaved changes label when in an unsaved state of saved search ({kibana-pull}169548[#169548]).
+* Allows changing the current sample size and saving it with a saved search ({kibana-pull}157269[#157269]).
+* Adds new sparse vector and dense vector icons ({kibana-pull}169493[#169493]).
+* Adds `sparse_vector` field support ({kibana-pull}168186[#168186]).
+Elastic Security::
+For the Elastic Security 8.12.0 release information, refer to {security-guide}/release-notes.html[_Elastic Security Solution Release Notes_].
+Elastic Search::
+*  Split details panel from model selection list ({kibana-pull}173434[#173434]).
+Fleet::
+* Adds support for Elasticsearch output performance presets ({kibana-pull}172359[#172359]).
+* Adds a new `keep_monitoring_alive` flag to agent policies ({kibana-pull}168865[#168865]).
+* Adds support for additional types for dynamic mappings ({kibana-pull}168842[#168842]).
+* Implements Elastic Agent upgrade states UI ({kibana-pull}167539[#167539]).
+* Use default component templates from Elasticsearch ({kibana-pull}163731[#163731]).
+Lens & Visualizations::
+* Allows coloring an entire metric panel when applying a maximum value to the metric visualization in **Lens** ({kibana-pull}172531[#172531]).
+* Adds truncation for data view pickers and field lists with many characters ({kibana-pull}172296[#172296]).
+* Allows searching in the documentation description ({kibana-pull}171916[#171916]).
+* Allows setting non-numeric metrics for metric visualizations in *Lens* ({kibana-pull}169258[#169258]).
+* Moves the tagcloud visualization in *Lens* out of experimental status ({kibana-pull}168824[#168824]).
```

Comment:
Can we move it as the first item of this list?

Reply:
Yeah no problem, updated in latest commit